src/cmd-remote-build-container

```diff
@@ -132,6 +134,8 @@ def main():
     # Check for requisite env vars
     if os.environ.get('CONTAINER_HOST') is None or os.environ.get('CONTAINER_SSHKEY') is None:
         sys.exit('You must have CONTAINER_HOST and CONTAINER_SSHKEY environment variables setup')
+    if args.write_digest_to_file is not None and not args.push_to_registry:
+        sys.exit('argument --write-digest-to-file can only be used with --push-to-registry')
```

Comment:
I couldn't find an easy way to express this using more argparse-native APIs instead.

Reply:
There this file will be? in the remote or in the host? Don't we need something  like the  `pull_oci_archive_from_remote` ?

Reply:
It will be on the host. It seems like podman is smart enough in this case to do the pushing on the remote, but the writing of this file on the host.